www/source/images/habitat-logo-birthday.svg

```diff
@@ -1 +1,80 @@
7.16A1.9,1.9,0,0,0,75.06,48v7.61"/><path class="cls-5" d="M80.85,46.41H77.16A1.9,1.9,0,0,0,75.06,48v7.61h3.69V48A1.9,1.9,0,0,1,80.85,46.41Z"/><path class="cls-6" d="M104.72,48v2.16a5.3,5.3,0,0,1-5.93,0,5.3,5.3,0,0,1-5.93,0,5.3,5.3,0,0,1-5.93,0,5.3,5.3,0,0,1-5.93,0,4.86,4.86,0,0,1-2.24.86,5.21,5.21,0,0,1-3.69-.86V48a1.9,1.9,0,0,1,2.1-1.59h25.47A1.9,1.9,0,0,1,104.72,48Z"/><path class="cls-7" d="M80.85,46.41A1.9,1.9,0,0,0,78.75,48v3a5.21,5.21,0,0,1-3.69-.86V48a1.9,1.9,0,0,1,2.1-1.59Z"/><path class="cls-8" d="M105.62,55.54H74.16V57a1,1,0,0,0,1,1h29.54a1,1,0,0,0,1-1v-1.5Z"/><path class="cls-9" d="M77.6,55.54H74.16V57a1,1,0,0,0,1,1H77.6Z"/><circle class="cls-4" cx="78.03" cy="48.18" r="0.56"/><circle class="cls-4" cx="83.97" cy="48.18" r="0.56"/><circle class="cls-4" cx="89.91" cy="48.18" r="0.56"/><circle class="cls-4" cx="95.86" cy="48.18" r="0.56"/><circle class="cls-4" cx="101.8" cy="48.18" r="0.56"/><path class="cls-6" d="M88,22.6a4,4,0,0,1,1.89-3.3,4,4,0,0,1,1.89,3.3,2,2,0,1,1-3.78,0Z"/><path class="cls-7" d="M89.89,24.21a.35.35,0,0,0-.35.35v1.89h.71V24.56A.35.35,0,0,0,89.89,24.21Z"/><path class="cls-4" d="M92.58,40.95a.71.71,0,0,1-.71-.71v-14h-4a1.83,1.83,0,0,1-1.83,1.83v2.33h1.83v9.82a.71.71,0,0,1-.71.71H86.09v2.29h7.61V40.95Z"/><path class="cls-10" d="M185.2,49.34a5.5,5.5,0,0,1-.4,1.08,5,5,0,0,1-.25.46.4.4,0,0,1-.67-.44,4.74,4.74,0,0,0,.54-1.31,2.21,2.21,0,0,0-.5-2.21,3,3,0,0,1-.7-3,5.41,5.41,0,0,1,.32-.92h.9a4.75,4.75,0,0,0-.44,1.13,2.2,2.2,0,0,0,.49,2.21A3,3,0,0,1,185.2,49.34Z"/><path class="cls-10" d="M181.4,43.52h1.12l1.12-3.75S181.4,41.66,181.4,43.52Z"/><path class="cls-11" d="M183.65,39.77s2.25,1.87,2.25,3.75h-3.37A8.46,8.46,0,0,1,183.65,39.77Z"/><path class="cls-10" d="M161.55,11.79c0,16.38,21.75,28.33,22.09,28.52L174.38,0q-.51,0-1,0A11.79,11.79,0,0,0,161.55,11.79Z"/><path class="cls-11" d="M183.65,6.08a11.78,11.78,0,0,1,22.1,5.71c0,16.51-22.1,28.52-22.1,28.52-.31-.21-18.08-12.15-18.08-28.52,0-6.17,3.87-11.23,8.81-11.74A11.78,11.78,0,0,1,183.65,6.08Z"/><path class="cls-1" d="M196,19.55l-3,3a.46.46,0,0,1-.79-.33V21.14a.46.46,0,0,0-.46-.46h-1.63a.46.46,0,0,0-.46.46v2.13a.46.46,0,0,0,.46.46h.55a.46.46,0,0,1,.33.79l-3,3a.46.46,0,0,1-.64,0l-3-3a.46.46,0,0,1,.33-.79h.64a.46.46,0,0,0,.46-.46v-2.1a.46.46,0,0,0-.46-.46h-3.57a.46.46,0,0,0-.46.46v2.1a.46.46,0,0,0,.46.46h.67a.46.46,0,0,1,.33.79l-3,3a.46.46,0,0,1-.64,0l-3-3a.46.46,0,0,1,.33-.79h.48a.46.46,0,0,0,.46-.46V21.14a.46.46,0,0,0-.46-.46h-1.61a.46.46,0,0,0-.46.46v1a.46.46,0,0,1-.79.33l-3-3a.46.46,0,0,1,0-.64l3-3a.46.46,0,0,1,.79.33v1a.46.46,0,0,0,.46.46h1.58a.46.46,0,0,0,.46-.46V15.16a.46.46,0,0,0-.46-.46h-.53a.46.46,0,0,1-.33-.79l3-3a.46.46,0,0,1,.64,0l3,3a.46.46,0,0,1-.33.79h-.67a.46.46,0,0,0-.46.46v2.08a.46.46,0,0,0,.46.46h3.57a.46.46,0,0,0,.46-.46V15.15a.46.46,0,0,0-.46-.46h-.64a.46.46,0,0,1-.33-.79l3-3a.46.46,0,0,1,.64,0l3,3a.46.46,0,0,1-.33.79h-.55a.46.46,0,0,0-.46.46v2.13a.46.46,0,0,0,.46.46h1.63a.46.46,0,0,0,.46-.46v-1a.46.46,0,0,1,.79-.33l3,3A.46.46,0,0,1,196,19.55Z"/></g></g></svg>
```

Comment:
I'm not seeing where this logo is used (but it does look like `habitat-birthday-dark` is, for example here: http://localhost:4567/blog/2017/06/Happy-Birthday-Habitat).

Reply:
Good catch!